### DISCUSSION TITLE
Allow checking on CI

### DISCUSSION THREAD

**Discussion by ghost** [sapegin/mrm#300] [Ideas]
It would be helpful if mrm allowed for a `--check` command, [like prettier does](https://prettier.io/docs/en/cli.html#check). The command should exit with an error if the current project files are not up-to-date with the config (so if there should be any changes made to the project files).

That way, a user can:

1. set their preferred mrm preset as a dependency for their project
2. add a script to check on CI: `"mrm:check": "mrm gitignore license --check --preset my-preset"`

That way, they'll be notified by a failing build/test if their project is not up to date with their mrm settings. It'll allow for easier updating with tools like greenkeeper as well.

**Comment by sapegin** [comment]
Seems like an okay feature. Feel free to send a pull request!

**Comment by ghost** [comment]
That'll probably be a while, but if I have the time I will! 👍
